modules/debts_api/app/controllers/debts_api/v0/financial_status_reports_controller.rb

```diff
@@ -23,6 +24,27 @@ def download_pdf
         )
       end
 
+      def submissions
+        submissions = DebtsApi::V0::Form5655Submission.where(user_uuid: current_user.uuid)
+        render json: { 'submissions' => submissions.map { |sub| { 'id' => sub.id } } }
+      end
+
+      def rehydrate
+        submission_id = params[:submission_id]
+
+        DebtsApi::V0::FsrRehydrationService.attempt_rehydration(user_uuid: current_user.uuid, submission_id:)
```

Comment:
Is `submission_id:` missing a value?

Reply:
Nope! Ruby has a fun little bit of syntactic sugar where you don't need to specify a value for hash key-value pairs if the key and value are the same. In this case line 33 defines `submission_id` in scope so we don't need to specify `submission_id: submission_id`. If you want to make doubly sure that it works, check out `modules/debts_api/spec/request/debts_api/v0/financial_status_reports_controller_spec.rb`